src/view.js

```diff
@@ -300,6 +327,28 @@ Marionette.View = Backbone.View.extend({
     }, children);
   },
 
+  // Internal utility for building an ancestor
+  // view tree list.
+  _getAncestors: function() {
+    var ancestors = [];
+    var parent  = this._parent;
+
+    while (parent) {
+      ancestors.push(parent);
+      parent = parent._parent;
```

Comment:
Since this chain can be quite large for complex apps and this is a "hot" method, should we investigate a recursive trampolined implementation?